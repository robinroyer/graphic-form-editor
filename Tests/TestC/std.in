CLEAR
R r01 10 10 5 5
C c01 10 10 5
L l01 10 10 5 5
PL pl01 10 10 5 5 3 3
CLEAR
EXIT