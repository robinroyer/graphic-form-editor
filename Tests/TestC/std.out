OK
#Objects cleared
OK
#New object: r01
OK
#New object: c01
OK
#New object: l01
OK
#New object: pl01
OK
#Objects cleared
